package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bs-react-router",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
1. I would avoid bumping the version in a PR, it should be done separately
2. it should be a `minor` bump, not a `patch`

Reply:
Definitely !! @bassjacob this is not a `patch`, this is a `minor` version cause of this [change](https://github.com/reasonml-community/bs-react-router/pull/6/files#diff-25e9ff6df1e15d2ff1a73808e358c3b5R68) from this comment https://github.com/reasonml-community/bs-react-router/pull/6#discussion_r160025614